lib/Support/ModuleInterpreter.cpp

```diff
@@ -450,17 +461,19 @@ void ModuleInterpreter::value_to_disk(const std::string &filename,
                                       const std::string &name,
                                       std::vector<float> &data,
                                       bool express_type) {
-  // auto value = value_map.at(name);
-  // if (express_type && module::isState(module::State::TPU_LOWERED)) {
-  //   if (module::isUniformQuantized(value)) {
-  //     auto qtype = module::getUniformQuantizedType(value);
-  //     for (auto &d : data) {
-  //       d = (d - (float)qtype.getZeroPoint()) * (float)qtype.getScale();
-  //     }
-  //   }
-  // }
-  // cnpy::npz_save(filename, name, data, "a");
-  llvm_unreachable("Not Implemented");
+   auto value = value_map.at(name);
+   if (express_type && module::isState(module::State::TPU_LOWERED)) {
+     if (module::isUniformQuantized(value)) {
+       auto qtype = module::getUniformQuantizedType(value);
+       for (auto &d : data) {
+         d = (d - (float)qtype.getZeroPoint()) * (float)qtype.getScale();
+       }
+     }
+   }
+   if(store_disk_shape.cbegin()!=store_disk_shape.cend())
+   store_disk_shape.clear();
+   store_disk_shape.push_back(data.size());
+   cnpy::npz_save(filename,name,&data[0],store_disk_shape,"a");
```

Comment:
```suggestion
   cnpy::npz_save(filename,name,&data[0],"a");
```
This should be fine.

Reply:
This is indeed a more concise API, but using it directly can lead to serious errors,I tried to take time to understand, but there seems to be little information available, so I chose to use the current API.Here is the error tracking information：
`File "/workspace/tpu-mlir/python/tools/model_deploy.py", line 18, in <module>
    import pymlir
ImportError: /workspace/tpu-mlir/install/python/pymlir.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZN4cnpy8npz_saveIfEEvNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES6_RKSt6vectorIT_SaIS8_EES6_
`